client/components/event-details/SimilarEvents.tsx

```diff
@@ -0,0 +1,54 @@
+import styled from 'styled-components'
+import { Id, Event } from '../../context'
+import { desktopFontSize, fontWeight, largerThan, mobileFontSize, tablet } from '../../styles'
+import { EventCard } from '../../components/club-details'
+
+const Title = styled.p`
+  font-size: ${mobileFontSize.h2};
+  font-weight: ${fontWeight.semiBold};
+
+  @media ${largerThan(tablet)} {
+    font-size: ${desktopFontSize.h2};
+  }
+`
+
+const tagsEqual = (a: Array<string>, b: Array<string>) => {
+  const aSet = new Set(a)
+  const bSet = new Set(b)
+
+  return aSet.size === bSet.size && [...aSet].every((value) => bSet.has(value))
```

Comment:
weird how awesome a lot of the new ES syntax is, but then we still have to do crazy stuff like this to check set equality